hphp/hack/src/errors/nast_check_error.ml

```diff
@@ -445,7 +446,30 @@ let illegal_context pos name =
         Naming_special_names.Coeffects.contexts )
     []
 
-let case_fallthrough switch_pos case_pos =
+let case_fallthrough switch_pos case_pos next_pos =
+  let rec addspaces fallthrough_comment num_spaces =
+    if num_spaces > 0 then
+      addspaces (fallthrough_comment ^ " ") (num_spaces - 1)
+    else
+      fallthrough_comment
+  in
+
+  let quickfixes =
+    match next_pos with
+    | None -> []
+    | Some next_pos ->
+      let (_, start_col) = Pos.line_column next_pos in
+      let new_pos = Pos.set_col_end (start_col - 5) next_pos in
+      let new_text = addspaces "  // FALLTHROUGH\n" (start_col - 5) in
```

Comment:
Why the `- 5` here?

Reply:
We only have the position for what the case is matching on, so I moved the start column 5 left to before "case "

Reply:
Gotcha. Rather than using a 'magic constant' where the reader has to work that out, this would be clearer as `String.length "case: "`.